src/styles/product_card.scss

```
@@ -0,0 +1,118 @@
+
+$cardWidth: 200px;
+$gapX: 40px;
+$paddingX: 40px;
+$colums: 4;
+
+.maincon {
+  margin: auto;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 0;
+  padding: 46px 26px 46px 28px;
+  width: $colums * $cardWidth + ($colums - 1) * $gapX + 2 * $paddingX;
+
+  &__product-card {
+    margin-right: 48px;
+    margin-bottom: 48px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-last-child(-n + 4) {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.cards {
+  width: 200px;
+  height: 406px;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+
+  &__photo {
+    margin: 0 3px 36.5px;
+
+    &_product {
+      width: 100%;
+      height: 134px;
+    }
+  }
```

It should be separate elements
```suggestion
  &__photo {
    margin: 0 3px 36.5px;
  }
  
  &_product {
      width: 100%;
      height: 134px;
    }
```

---

I have class="cards__photo_product"